pkg/autodiscovery/common/types/prometheus.go

```diff
@@ -115,6 +149,17 @@ func (pc *PrometheusCheck) Init() error {
 
 // initInstances defaults the Instances field in PrometheusCheck
 func (pc *PrometheusCheck) initInstances() {
+	var openmetricsDefaultMetrics []string
+	switch config.Datadog.GetInt("prometheus_scrape.version") {
+	case 1:
+		openmetricsDefaultMetrics = OpenmetricsDefaultMetricsV1
+	case 2:
+		openmetricsDefaultMetrics = OpenmetricsDefaultMetricsV2
+	default:
+		log.Errorf("Invalid value for `prometheus_scrape.version`. Can only be 1 or 2.")
```

Comment:
Nit: It can be useful to print the value returned by `config.Datadog.GetInt("prometheus_scrape.version")` in this error log.

Reply:
Done!